public/app/plugins/datasource/prometheus/components/PromQueryEditor.tsx

```diff
@@ -139,27 +164,33 @@ export class PromQueryEditor extends PureComponent<PromQueryEditorProps, State>
 
             <div className="gf-form">
               <InlineFormLabel
-                width={7}
+                width={5}
                 tooltip={
                   <>
-                    An additional lower limit for the step parameter of the Prometheus query and for the{' '}
-                    <code>$__interval</code> and <code>$__rate_interval</code> variables. The limit is absolute and not
-                    modified by the &quot;Resolution&quot; setting.
+                    Optionally set lower or upper bounds on the interval between datapoints, e.g., set &quot;minimum
```

Comment:
@achatterjee-grafana could you look at this copy. 🙂

Reply:
How about: "Optionally, set the lower or upper bounds on the interval between data points, for example, set &quot; minimum."

If you like, make the same change in other areas of the code as needed. Thanks!